pkg/ccl/backupccl/backup_test.go

```diff
@@ -9167,12 +9165,14 @@ func TestRestorePauseOnError(t *testing.T) {
 			})
 	}
 
+	t.Logf("Test")
+	sqlDB.Exec(t, `SET CLUSTER SETTING jobs.debug.pausepoints = 'restore.after_restore_failure'`)
```

Comment:
IMO you could delete the whole test?

Reply:
do you think it's worth leaving the test for now until we know the CombineErrors is fine in master, or are you confident enough that we can drop it now

Reply:
I would probably just delete it now; we don't maintain tests for other pause point calls.